test/dleq.test.ts

```diff
@@ -3,9 +3,12 @@
 // Licensed under the BSD-3-Clause license found in the LICENSE file or
 // at https://opensource.org/licenses/BSD-3-Clause
 
-import { DLEQParams, DLEQProof, DLEQProver, Elt, Group, Scalar } from '../src/index.js'
+import { DLEQParams, DLEQProof, DLEQProver, Elt, Scalar } from '../src/index.js'
 
 import { serdeClass } from './util.js'
+import { GroupConsSjcl } from '../src/groupSjcl.js'
+
+const Group = GroupConsSjcl
```

Comment:
```suggestion
import { Group } from '../src/index.js'
```

Reply:
and similarly to the other test files.

Reply:
I had actually planned to remove all these from the .test.ts files and just configure the Group inside src/index.js ? So the tests would not have to import anything ?

Reply:
Ok, so here they are using the Group. I think it's better to simply access Oprf.Group to get the Group, that way it mirrors the setter ? You must do `Oprf.Group = SomeCons` to set the group? What do you think? It feels like it would be better to have only one way to get the group ?

Reply:
For the sole case of accessing the group to do group operations, then getting group from "voprf-ts/index.js" makes more sense.

If we want to use the specific group from an Oprf instance, then `Oprf.Group` makes sense in this case.

Having said that, can we have both options?

Reply:
> Having said that, can we have both options?

We can, but I am less sure than you it makes sense.

> For the sole case of accessing the group to do group operations, then getting group from "voprf-ts/index.js" makes more sense.

Consider:

The Group is a static member of Oprf. Only one Group implementation can be active at a time in the library, unless we refactor everything to an api factory (doable, but probably over-engineering). Getting/setting in one place makes that more apparent. 

Other than the tests what other client for the Group is expected ?

Reply:
I was trying to look ahead on the implementation of, for example, opaque-ts. But I think, we can wait until that use case emerges by itself.